### DISCUSSION TITLE
SQL Editor File does not show state as SAVED when there are uncommitted transactions

### DISCUSSION THREAD

**Discussion by jdesmet** [dbeaver/dbeaver#19851] [Q&A]
### Description

When the SQL Editor thinks that there are Uncommitted transactions, it will always show the file as Unsaved (no matter how many times you try to save the file). Also I do not know why I have Uncommitted transactions if  all what I ran were SELECT statements. It appears as if DBeaver considers OPEN CURSORS also as UNCOMMITTED TRANSACTIONS?

### DBeaver Version

Version 23.0.2.202304091457

**Comment by E1izabeth** [comment]
Such indication is intentional to show that work in SQL editor is not finished.

If DBeaver consider that transaction is opened depends on the transaction mode you use.
